server/db/db.js

```diff
@@ -1262,7 +1262,7 @@ async function getRecentSensorsVitals(pgClient) {
       WHERE d.device_type = $1
       ORDER BY sv.created_at
       `,
-      [DEVICE_TYPE.DEVICE_SENSOR],
+      [DEVICE_TYPE.DEVICE_SENSOR_SINGLESTALL],
```

Comment:
likewise, for test cases, u will want to test some inputs with multistall and make the others compatible with both multi and single stalls.